src/problem/func/func_SLIM_VMAG.c

```diff
@@ -216,11 +216,11 @@ void FUNC_SLIM_VMAG_eval_step(Func* f, Branch* br, int t, Vec* var_values) {
 
     if (!bus_counted[bus_index_t[k]]) {
 
-      dv = BUS_get_v_max(bus)-BUS_get_v_min(bus);
+      dv = BUS_get_v_max_reg(bus)-BUS_get_v_min_reg(bus);
       if (dv < FUNC_SLIM_VMAG_PARAM)
 	dv = FUNC_SLIM_VMAG_PARAM;
 
-      vmid = 0.5*(BUS_get_v_max(bus)+BUS_get_v_min(bus));
+      vmid = 0.5*(BUS_get_v_max_reg(bus)+BUS_get_v_min_reg(bus));
```

Comment:
Yes, right now the C functions do not have documentation. However, they are documented in the python docs with math in many cases:
http://pfnet-python.readthedocs.io/en/latest/problems.html#voltage-magnitude-soft-limit-penalty